bootstrap/bootstrap.php

```diff
@@ -10,12 +10,7 @@
 $request = $app->router->match($GLOBALS, $_SERVER);
 
 try {
-    $page = $app->resource
-        ->{$request->method}
-        ->uri($request->path)
-        ->withQuery($request->query)
-        ->eager
-        ->request();
+    $page = $app->resource->{$request->method}->uri($request->path)($request->query);
```

Comment:
Yes, it returns invokable object.
https://github.com/bearsunday/BEAR.Resource/blob/1.x/src/AbstractRequest.php#L136


// lazy request (You can assign to template, then it evaluates on appeared. Ideal in if condition in template.)

```php
$request =  $app->resource->get->uri('app://sef/user');
```

// eager request
```php
$request =  $app->resource->get->uri('app://sef/user')();
```